roles/kubernetes-apps/csi_driver/cinder/templates/cinder-csi-driver.yml.j2

```diff
@@ -4,4 +4,4 @@ metadata:
   name: cinder.csi.openstack.org
 spec:
   attachRequired: true
-  podInfoOnMount: false
+  podInfoOnMount: true
```

Comment:
As I see in the documentation, this is used to add specific tags to the volumes. It is not set in the Cinder CSI doc: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-cinder-csi-plugin.md
I don't know what is it used for?

Reply:
It enables inline volumes:

> This feature allows CSI volumes to be directly embedded in the Pod specification instead of a PersistentVolume. Volumes specified in this way are ephemeral and do not persist across Pod restarts. As of Kubernetes v1.16 this feature is beta so enabled by default. To enable this feature for CSI Driver, volumeLifecycleModes needs to be specified in CSIDriver object. The driver can run in Persistent mode, Ephemeral or in both modes. podInfoOnMount must be true to use this feature.